### DISCUSSION TITLE
Clone Y axis

### DISCUSSION THREAD

**Discussion by Omar5x** [fra589/grbl-Mega-5X#324] [Q&A] [answered]
Hi,first of all i want to thank you for making this programme and making it public it's realyy cool. So I'm building a 4 axis tufting machine to make rugs I'll be using 1 stepper for the Z axis
1 for the X axis and 1 for the A axis to turn the tufting gun,and for the Y axis i want to use two stepper motors the y axis that I'm already using plus i want to use E1 driver. Is it possible and if yes how can i do it.
Thank you

**Comment by RaphaelDives** [comment, marked as answer]
Hello @Omar5x,
welcome to grbl-mega5x! 😄 
For cloning you just have to set the axis names in config.h!

Please refer to:
https://github.com/fra589/grbl-Mega-5X/wiki/Quick-Guide-to-Setting-Up-Your-Machine-for-the-First-Time#cloning

If you encounter any problems, you are welcome! 😊

Cheers
Raphael

**Reply by Omar5x** [reply]
Thanks❤️❤️
